src/pages/about.vue

```diff
@@ -9,18 +9,9 @@
 </template>
 
 <script>
-import AboutPage from '~/components/AboutPage/AboutPage';
-import PageHeader from '~/components/PageHeader';
-import SiteNavBar from '~/components/SiteNavBar';
-
-import { CrispMixinFactory } from '~/mixins/crisp';
-
 export default {
-  components: {
-    AboutPage,
-    PageHeader,
-    SiteNavBar,
+  mounted() {
+    window.location.href = 'https://about.like.co/';
```

Comment:
Please use `fetch(context)` instead of `mounted()` life cycle with `context.redirect(URL)` and remove everything else

Reply:
wow, so cool 👀, It's much simpler ~